config/initializers/differ.rb

```diff
@@ -0,0 +1,6 @@
+require_dependency 'differ/differ'
+require_dependency 'differ/simple_diff'
+
+if ENV['DIFFER_SOURCE']
+  Differ.register(:source, Differ::SimpleDiff.new(ENV['DIFFER_SOURCE']))
```

Comment:
> allow arbitrary many differs with arbitrary names to be configured through ENV by picking up all vars with names that match `DIFFER_*`?

Ha! I thought about the same idea and then said “nah, let’s not get ahead of ourselves.” I think I would at least change the var to `DIFFER_SIMPLE_*` if we did that.

I can imagine some differs being more complex (or maybe `SimpleDiff` will just grow)—we can‘t meaningfully diff PDFs with this, for example, but we really should eventually have that capability. There’s a lot of PDF content that is “tracked” but has no real tooling to support analyzing it. Versionista is also hitting some audio files and potentially things like CSVs. I haven't really done a full audit of content types. By nature, I’m sure PF is grabbing who-knows-what variety of content types.